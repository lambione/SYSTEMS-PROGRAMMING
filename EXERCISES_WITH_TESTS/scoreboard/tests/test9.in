aaaaaaaaaaaaaaaaaaaa bbbbbbbbbbbbbbbbbbbb 0 0
cccccccccccccccccccc dddddddddddddddddddd 0 0
eeeeeeeeeeeeeeeeeeee ffffffffffffffffffff 0 0
gggggggggggggggggggg hhhhhhhhhhhhhhhhhhhh 0 0
iiiiiiiiiiiiiiiiiiii jjjjjjjjjjjjjjjjjjjj 0 0
