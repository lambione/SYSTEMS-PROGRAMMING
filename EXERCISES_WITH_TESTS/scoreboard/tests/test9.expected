aaaaaaaaaaaaaaaaaaaa 1 1
bbbbbbbbbbbbbbbbbbbb 1 1
cccccccccccccccccccc 1 1
dddddddddddddddddddd 1 1
eeeeeeeeeeeeeeeeeeee 1 1
ffffffffffffffffffff 1 1
gggggggggggggggggggg 1 1
hhhhhhhhhhhhhhhhhhhh 1 1
iiiiiiiiiiiiiiiiiiii 1 1
jjjjjjjjjjjjjjjjjjjj 1 1
